consensus/src/quorum_store/batch_coordinator.rs

```diff
@@ -47,35 +44,23 @@ impl BatchCoordinator {
 
     async fn handle_batch(&mut self, batch: Batch) -> Option<PersistRequest> {
         let source = batch.author();
-        let expiration = batch.expiration();
         let batch_id = batch.batch_id();
         trace!(
             "QS: got batch message from {} batch_id {}",
             source,
             batch_id,
         );
-        if expiration.epoch() == self.epoch {
-            counters::RECEIVED_BATCH_COUNT.inc();
-            let num_bytes = batch.num_bytes();
-            if num_bytes > self.max_batch_bytes {
-                error!(
-                    "Batch from {} exceeds size limit {}, actual size: {}",
-                    source, self.max_batch_bytes, num_bytes
-                );
-                return None;
-            }
-            let persist_request = batch.into();
-            return Some(persist_request);
-        }
-        // Malformed request with an inconsistent expiry epoch.
-        else {
-            trace!(
-                "QS: got end batch message from different epoch {} != {}",
-                expiration.epoch(),
-                self.epoch
+        counters::RECEIVED_BATCH_COUNT.inc();
```

Comment:
I think it's still useful to isolate the batches on epochs, since the proof verification relies on epoch specific verifier